content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -182,6 +182,7 @@ For a reference to old feature gates that are removed, please refer to
 | `RemainingItemCount` | `true` | Beta | 1.16 | |
 | `RotateKubeletServerCertificate` | `false` | Alpha | 1.7 | 1.11 |
 | `RotateKubeletServerCertificate` | `true` | Beta | 1.12 | |
+| `RuntimeClassInImageCriApi` | `false` | Alpha | 1.29 | |
```

Comment:
Consider including a brief description of `RuntimeClassInImageCriApi` feature gate within the ["List of feature gates"](https://github.com/kubernetes/website/blob/dev-1.29/content/en/docs/reference/command-line-tools-reference/feature-gates.md#list-of-feature-gates-feature-gates) section on the same page. Ensure alphabetical order is maintained when adding it.